mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt

```diff
@@ -229,7 +231,9 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
 
     private fun applyLoadedBitmap(bitmap: Bitmap) {
         removeProgressDrawable()
-        if (adjustViewBoundsForDownscalingOnly) {
+        if (context.getPrefs().getBoolean(PrefKeys.IMAGE_UPSCALING, true)) {
```

Comment:
I don't like this pref evaluation here, since `WidgetImageView` is designed to be generic and doesn't know what kind of content it has.